docker-compose-auth.yml

```diff
@@ -7,46 +7,57 @@
 #    4. 9095: SASL over SSL
 #
 # See examples/auth for the associated cluster configs and certs.
-version: '2'
+version: '3'
 
 services:
   zookeeper:
-    image: "wurstmeister/zookeeper:latest"
+    container_name: zookeeper
+    hostname: zookeeper
+    image: bitnami/zookeeper:latest
     ports:
       - "2181:2181"
+    environment:
+      ALLOW_ANONYMOUS_LOGIN: yes
 
   kafka:
-    image: wurstmeister/kafka:2.13-2.7.1
+    container_name: kafka
+    hostname: kafka
+    image: bitnami/kafka:2.7.0
```

Comment:
Let's keep the Kafka version the same, we should use 2.7.1 here

Reply:
bitnami kafka does not have 2.7.1. Resolving this